joomla/joomla-cms
# [3.10] Make sure the extension install works as expected

### Summary of Changes

Only show a message when the verification fails (request by @mbabker )
Remove the check at install time.
Make sure we are using the correct update object and don't do some magic with passing the update xml.

### Testing Instructions

- Install 3.9-dev https://developer.joomla.org/nightlies/Joomla_3.9.0-dev-Development-Full_Package.zip
- Install any outdated extension that do not support hashes yet (well all 3rd party tools)
- notice the message saying that there is no hash provided
- install the update
- notice the message saying that there is no hash provided
- apply this patch
- redo the steps above
- notice that there is no message at all.
- install com_patchtester
- notice there is no message
- apply the update to the latest com_patchtester
- notice there is no message at all. (even as they use hashes)

### Expected result

Only show a message when the verification fails (request by @mbabker )
Remove the check at install time.
Make sure we are using the correct update object and don't do some magic with passing the update xml.

### Documentation Changes Required

None.
